backend/de.metas.business/src/main/java/de/metas/location/interceptor/C_Country.java

```diff
@@ -46,7 +47,8 @@
 @Component
 public class C_Country
 {
-	final CountryService countryService = SpringContextHolder.instance.getBean(CountryService.class);
+	private final CountryService countryService = SpringContextHolder.instance.getBean(CountryService.class);
+	private final int c_countryTableId = Services.get(IADTableDAO.class).retrieveTableId(I_C_Country.Table_Name);
```

Comment:
pls use `retrieveAdTableId` so that the `c_countryTableId` would be of type AdTableId

(basically avoid `int` IDs as much as possible)